doc/api/webstreams.md

```diff
@@ -1457,6 +1457,48 @@ added: v16.7.0
 * Returns: {Promise} Fulfills with an `ArrayBuffer` containing the full
   contents of the stream.
 
+```mjs
+import { arrayBuffer } from 'node:stream/consumers';
+import { Readable } from 'node:stream';
+
+// Creates a 1_000_000 items array of objects
+const items = Array.from(
+  {
+    length: 1e6
+  },
+  () => ({
+    message: 'hello world from consumers!'
+  })
+);
+
+const readable = Readable.from(JSON.stringify(items)); // Creates a readable stream from the array
+const data = await arrayBuffer(readable); // Consumes data stream as an arrayBuffer
+console.log(`from readable: ${data.byteLength}`); // From readable: 42000001
+```
+
+```cjs
+const { arrayBuffer } = require('node:stream/consumers');
+const { Readable } = require('node:stream');
+
+// Creates a 1_000_000 items array of objects
+const items = Array.from(
+  {
+    length: 1e6
+  },
+  () => ({
+    message: 'hello world from consumers!'
+  })
+);
+
+async function readFromStream() {
```

Comment:
Would keep it out of the function for better readability 
```js
;(async () => {
  const readable = Readable.from(JSON.stringify(items)); // Creates a readable stream from the array
  const data = await arrayBuffer(readable); // Consumes data stream as an arrayBuffer
  console.log(`from readable: ${data.byteLength}`); // From readable: 42000001
})();
```

or

```js
const readable = Readable.from(JSON.stringify(items)); // Creates a readable stream from the array
arrayBuffer(readable).then(data => {
  console.log(`from readable: ${data.byteLength}`); // From readable: 42000001
})

```

Reply:
Yeah that was my first iteration, but I didn't want to leave IIAE's because I think it makes the code a bit more difficult to read, I think I'll go with the promise option